schema-generator/src/main/java/com/github/jrcodeza/schema/generator/OpenAPIGenerator.java

```diff
@@ -196,7 +194,12 @@ private Paths createPathsWrapper() {
     }
 
     private Map<String, PathItem> createPathExtensions() {
-        ClassPathScanningCandidateComponentProvider scanner = new ClassPathScanningCandidateComponentProvider(false);
+		ClassPathScanningCandidateComponentProvider scanner;
```

Comment:
Create method for this so we don't have if else structure for initialization inside a method that does other things based on name

Reply:
done, thanks